### ISSUE TITLE
Since v0.1.4 all storage writes have gone to a single folder

### ISSUE THREAD

**Issue by NickJH** [SiaFoundation/hostd#148]
### Current Behavior

Since v0.1.4 all storage writes have gone to a single folder
```
root@hostd-test:/var/lib/hostd-data# ll
total 2211047056
drwxr-xr-x  2 root root          4096 Jul 22 11:47 ./
drwxr-xr-x 48 root root          4096 Jun  4 08:54 ../
-rw-r--r--  1 root root 1999999336448 Aug 14 09:10 data1.dat
-rw-r--r--  1 root root 1999999336448 Aug  8 05:15 data2.dat
-rw-r--r--  1 root root 1999999336448 Aug  8 05:13 data3.dat
-rw-r--r--  1 root root 1999999336448 Aug  8 05:15 data4.dat
```

### Expected Behavior

Storage should be distributed between folders

### Version

v0.1.4

### What operating system did the problem occur on (e.g. Ubuntu 22.04, macOS 12.0, Windows 11)?

Ubuntu 22.04

### Anything else?

This was working with v0.1.4-beta1 which I was running from 02 Aug to 08 Aug

**Comment by artur9010** [comment]
Looks like I have same issue. 1GB is moved to another volume by shrinking first one (#149)

![image](https://github.com/SiaFoundation/hostd/assets/4835677/e6f70767-997a-400d-8bfd-38a93de2430e)

**Comment by n8maninger** [comment]
Fixed by #150
